symphony-bdk-core/src/main/java/com/symphony/bdk/core/SymphonyBdk.java

```diff
@@ -134,6 +137,15 @@ public PresenceService presences() {
     return this.presenceService;
   }
 
+  /**
+   * Get the {@link ConnectionService} from a Bdk entry point.
+   *
+   * @return {@link ConnectionService} connection service instance.
+   */
+  public ConnectionService connections() {
```

Comment:
Should be available from https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/master/symphony-bdk-core/src/main/java/com/symphony/bdk/core/OboServices.java as well (same for `PresenceService`)